tests/functional/test_install_vcs_svn.py

```diff
@@ -4,9 +4,12 @@
 from pip._internal.vcs.subversion import Subversion
 
 
+@patch('pip._internal.vcs.subversion.Subversion.get_remote_call_options')
 @patch('pip._internal.vcs.call_subprocess')
 @pytest.mark.network
-def test_obtain_should_recognize_auth_info_url(call_subprocess_mock, script):
+def test_obtain_should_recognize_auth_info_url(
+        call_subprocess_mock, get_remote_call_options_mock, script):
+    get_remote_call_options_mock.return_value = []
```

Comment:
If you instantiate `Subversion` with `use_interactive=False`, you can feed two birds with one scone and both (1) avoid having to mock `get_remote_call_options()` and (2) test that remote call options are actually used.

Reply:
> feed two birds with one scone 

🤣

Reply:
Great idea, I've pushed in the update.